code/game/gamemodes/vampire/grave_fever.dm

```diff
@@ -0,0 +1,32 @@
+/datum/disease/vampire
+	name = "Grave Fever"
+	max_stages = 3
+	stage_prob = 5
+	spread_text = "Non-Contagious"
+	spread_flags = SPECIAL
+	cure_text = "Antibiotics"
+	cures = list("spaceacillin")
+	agent = "Grave Dust"
+	cure_chance = 20
+	viable_mobtypes = list(/mob/living/carbon/human)
+	severity = DANGEROUS
+	disease_flags = CURABLE
+	spread_flags = NON_CONTAGIOUS
+
+/datum/disease/vampire/stage_act()
+	..()
+	var/toxdamage = stage * 2
```

Comment:
mob life tick is around 2 seconds
mobs have 100 health before crit
tox damage at stage 5 is 10
5 damage at 15% chance per second

100 health / [5 * .15] = crit in 2 minutes at stage 5 on average.

just puttin the math out there.